test/server/models/Update.test.js

```diff
@@ -195,16 +198,60 @@ describe('server/models/Update', () => {
         const usersToNotify = await update.getUsersToNotify();
         const receivedEmails = usersToNotify.map(u => u.email);
 
-        const expectAllEmailsFrom = usersList => {
-          return usersList.forEach(user => expect(receivedEmails).to.include(user.email));
-        };
-
-        expectAllEmailsFrom(parentCollectiveAdmins);
-        expectAllEmailsFrom(collectiveAdmins);
-        expectAllEmailsFrom(individualBackersUsers);
-        expectAllEmailsFrom(getOrganizationAdminUsers());
+        expectAllEmailsFrom(parentCollectiveAdmins, receivedEmails);
+        expectAllEmailsFrom(collectiveAdmins, receivedEmails, receivedEmails);
+        expectAllEmailsFrom(individualBackersUsers, receivedEmails);
+        expectAllEmailsFrom(getOrganizationAdminUsers(), receivedEmails);
         expect(usersToNotify.length).to.eq(expectedPrivateTotal);
       });
+
+      it('Notifies child collective users when parent collective public update is made', async () => {
+        collective.remove;
+        const update = await fakeUpdate({ CollectiveId: parentCollective.id, isPrivate: false });
+        const usersToNotify = await update.getUsersToNotify();
+        const receivedEmails = usersToNotify.map(u => u.email);
+
+        expectAllEmailsFrom(parentCollectiveAdmins, receivedEmails);
+        expectAllEmailsFrom(parentCollectiveBackers, receivedEmails);
+        expectAllEmailsFrom(parentCollectiveFollowers, receivedEmails);
+        expectAllEmailsFrom(individualBackersUsers, receivedEmails);
+        expectAllEmailsFrom(getOrganizationAdminUsers(), receivedEmails);
+
+        // Number of collective admins who are backers
+        const numberOfCollectiveAdminsAddedToBackers = 2;
```

Comment:
Let's move that to a shared variable next to https://github.com/opencollective/opencollective-api/blob/07deff0b9b6f049caac10883448eab9f57705872/test/server/models/Update.test.js#L142, otherwise it's unclear where this number is coming from.